content/en/docs/devops-user-guide/how-to-use/pipeline-settings.md

```diff
@@ -1,9 +1,203 @@
+
+  {{</ notice >}}
+
+**Behavioral strategy**
+
+![behavioral-strategy](/images/docs/devops-user-guide/using-devops/pipeline-settings/behavioral-strategy.png)
+
+In **Behavioral strategy**, KubeSphere offers four strategies by default. As a Jenkins pipeline runs, the Pull Request (PR) submitted by developers will also be regarded as a separate branch.
+
+**Discover Branches**
+
+- **Exclude branches that are also filed as PRs**. The source branch is not scanned such as the origin’s master branch. These branches need to be merged.
+- **Only branches that are also filed as PRs**. Only scan the PR branch.
+- **All branches**. Pull all the branches from the repository origin.
+
+**Discover Tag Branches**
+
+- **Enable discovery of Tag branch**. The branch with a specific tag will be scanned.
+- **Disable the discovery of the Tag branch**. The branch with a specific tag will not be scanned.
+
+**Discover PR from Origin**
+
+- **Source code version of PR merged with target branch**. A pipeline is created and runs based on the source code after the PR is merged into the target branch.
+- **Source code version of PR itself**. A pipeline is created and runs based on the source code of the PR itself.
+- **Two pipelines are created when a PR is discovered**. KubeSphere creates two pipelines, one based on the source code after the PR is merged into the target branch, and the other based on the source code of the PR itself.
+
+**Discover PR from Forks**
+
+- **Source code version of PR merged with target branch**. A pipeline is created and runs based on the source code after the PR is merged into the target branch.
+- **Source code version of PR itself**. A pipeline is created and runs based on the source code of the PR itself.
+- **Two pipelines are created when a PR is discovered**. KubeSphere creates two pipelines, one based on the source code after the PR is merged into the target branch, and the other based on the source code of the PR itself.
+- **Contributors**. The users who make contributions to the PR.
+- **Everyone**. Every user who can access the PR.
+- **From users with Admin or Write permission**. Only from users with Admin or Write permission to the PR.
+- **Nobody**. If you select this option, no PR will be discovered despite the option you select in **Pull Strategy**.
+
+**Script Path**
+
+![script-path](/images/docs/devops-user-guide/using-devops/pipeline-settings/script-path.png)
+
+The field of **Script Path** specifies the Jenkinsfile path in the code repository. It indicates the repository’s root directory. If the file location changes, the script path also needs to be changed.
+
+**Scan Repo Trigger**
+
+![scan-repo-trigger](/images/docs/devops-user-guide/using-devops/pipeline-settings/scan-repo-trigger.png)
+
+You can check **Enable regular expressions, ignoring names that do not match the provided regular expression (including branches and PR, etc.)** to specify a regular expression as the trigger for scanning the code repository.
```

Comment:
(including branches and PR, etc.)
->
(including branches and PRs)

Change the UI text in the console repository.